magellan/sampler/down_sample.py

```diff
@@ -0,0 +1,235 @@
+                "cannot", "cant", "co", "con", "could", "couldnt", "cry", "de", "describe",
+                "detail", "do", "done", "down", "due", "during", "each", "eg", "eight",
+                "either", "eleven", "else", "elsewhere", "empty", "enough", "etc", "even",
+                "ever", "every", "everyone", "everything", "everywhere", "except", "few",
+                "fifteen", "fify", "fill", "find", "fire", "first", "five", "for", "former",
+                "formerly", "forty", "found", "four", "from", "front", "full", "further",
+                "get", "give", "go", "had", "has", "hasnt", "have", "he", "hence", "her",
+                "here", "hereafter", "hereby", "herein", "hereupon", "hers", "herself",
+                "him", "himself", "his", "how", "however", "hundred", "ie", "if", "in",
+                "inc", "indeed", "interest", "into", "is", "it", "its", "itself", "keep",
+                "last", "latter", "latterly", "least", "less", "ltd", "made", "many", "may",
+                "me", "meanwhile", "might", "mill", "mine", "more", "moreover", "most",
+                "mostly", "move", "much", "must", "my", "myself", "name", "namely",
+                "neither", "never", "nevertheless", "next", "nine", "no", "nobody",
+                "none", "noone", "nor", "not", "nothing", "now", "nowhere", "of", "off",
+                "often", "on", "once", "one", "only", "onto", "or", "other", "others",
+                "otherwise", "our", "ours", "ourselves", "out", "over", "own","part",
+                "per", "perhaps", "please", "put", "rather", "re", "same", "see", "seem",
+                "seemed", "seeming", "seems", "serious", "several", "she", "should",
+                "show", "side", "since", "sincere", "six", "sixty", "so", "some",
+                "somehow", "someone", "something", "sometime", "sometimes", "somewhere",
+                "still", "such", "system", "take", "ten", "than", "that", "the", "their",
+                "them", "themselves", "then", "thence", "there", "thereafter", "thereby",
+                "therefore", "therein", "thereupon", "these", "they", "thickv", "thin",
+                "third", "this", "those", "though", "three", "through", "throughout",
+                "thru", "thus", "to", "together", "too", "top", "toward", "towards",
+                "twelve", "twenty", "two", "un", "under", "until", "up", "upon", "us",
+                "very", "via", "was", "we", "well", "were", "what", "whatever", "when",
+                "whence", "whenever", "where", "whereafter", "whereas", "whereby",
+                "wherein", "whereupon", "wherever", "whether", "which", "while",
+                "whither", "who", "whoever", "whole", "whom", "whose", "why", "will",
+                "with", "within", "without", "would", "yet", "you", "your", "yours",
+                "yourself", "yourselves"
+    ]
+    return stop_words
+
```

Comment:
@Yashg19  why not put stop words list in utils ? So that other modules can use it. I think overlap blocker may also use stop words.